src/content/docs/en/reference/api-reference.mdx

```diff
@@ -2512,19 +2512,17 @@ export const server = {
 
 #### `handler()` property
 
-<p>
-<Since v="4.15.0" />
-</p>
+**Type:** `(input, context) => any`
+
+`defineAction()` requires a `handler()` function containing the server logic to run when the action is called. This function can return data that is automatically serialized and sent to the caller.
```

Comment:
```suggestion
`defineAction()` requires a `handler()` function containing the server logic to run when the action is called. This function returns data that is automatically serialized and sent to the caller.
```
Is there a reason it's only "can return"? Can it be more definitive. Or..

> When data is returned, it is automatically serialized and sent to the caller.

Reply:
Maybe even?

```suggestion
`defineAction()` requires a `handler()` function containing the server logic to run when the action is called. Data returned from the handler is automatically serialized and sent to the caller.
```